system_test/common/aest_channels_SUITE.erl

```diff
@@ -95,7 +95,8 @@
     name    => node2,
     peers   => [node1],
     backend => aest_docker,
-    source  => {pull, "aeternity/aeternity:local"}
+    source  => {pull, "aeternity/aeternity:local"},
+    mining => #{autostart => false}
```

Comment:
If both are mining simultaniously, using the fast mining setup, there is a constant struggle between the two nodes that produces a ton of race conditions.